src/libraries/System.IO.Hashing/tests/XxHash64Tests.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace System.IO.Hashing.Tests
+{
+    public class XxHash64Tests : NonCryptoHashTestDriver
+    {
+        private static readonly byte[] s_emptyHashValue =
+            new byte[] { 0xEF, 0x46, 0xDB, 0x37, 0x51, 0xD8, 0xE9, 0x99 };
+
+        public XxHash64Tests()
+            : base(s_emptyHashValue)
+        {
+        }
+
+        public static IEnumerable<object[]> TestCases
+        {
+            get
+            {
+                object[] arr = new object[1];
+
+                foreach (TestCase testCase in TestCaseDefinitions)
+                {
+                    arr[0] = testCase;
+                    yield return arr;
+                }
+            }
+        }
+
+        private const string ThirtyThreeBytes = "This string has 33 ASCII bytes...";
+        private const string ThirtyThreeBytes3 = ThirtyThreeBytes + ThirtyThreeBytes + ThirtyThreeBytes;
+        private const string DotNetNCHashing = ".NET now has non-crypto hashing";
+        private const string SixtyThreeBytes = "A sixty-three byte test input requires substantial forethought!";
+        private const string SixtyThreeBytes3 = SixtyThreeBytes + SixtyThreeBytes + SixtyThreeBytes;
+
+        protected static IEnumerable<TestCase> TestCaseDefinitions { get; } =
+            new[]
+            {
+                //https://asecuritysite.com/encryption/xxHash, Example 1
```

Comment:
These are unofficial test vectors?

Reply:
Yep.  They're the only static published `(input, output)` pairs that I found for the XxHashes.